uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/AttachmentsPopupElementView.qml

```diff
@@ -0,0 +1,212 @@
+                bottom: closeButton.top
+                left: parent.left
+                right: parent.right
+            }
+            fillMode: Image.PreserveAspectFit
+        }
+
+        Button {
+            id: closeButton
+            text: "Close"
+            anchors.bottom: parent.bottom
+            anchors.horizontalCenter: parent.horizontalCenter
+            onClicked: fullScreenImageDialog.visible = false;
+        }
+    }
+
+    MenuSeparator {
+        Layout.fillWidth: true
+        Layout.leftMargin: attachmentsPopupElementView.mediaMargin
+        Layout.rightMargin: attachmentsPopupElementView.mediaMargin
+    }
+
+    Column {
+        clip: true
+        focus: true
+        spacing: attachmentsPopupElementView.layoutSpacing
+
+        Label {
+            text: attachmentsPopupElementView.controller.title !== "" ? controller.title : "Attachments"
+            wrapMode: Text.WordWrap
+            font.pixelSize: 20
+            font.weight: Font.Black
+
+            Layout.fillWidth: true
+            Layout.leftMargin: attachmentsPopupElementView.mediaMargin
+            Layout.rightMargin: attachmentsPopupElementView.mediaMargin
+        }
+        Label {
+            text: attachmentsPopupElementView.controller.description
+            visible: attachmentsPopupElementView.controller.description !== ""
+            wrapMode: Text.WordWrap
+
+            Layout.fillWidth: true
+            Layout.leftMargin: attachmentsPopupElementView.mediaMargin
+            Layout.rightMargin: attachmentsPopupElementView.mediaMargin
+        }
+    }
+
+    MenuSeparator {
+        Layout.fillWidth: true
+        Layout.leftMargin: attachmentsPopupElementView.mediaMargin
+        Layout.rightMargin: attachmentsPopupElementView.mediaMargin
+    }
+
+    ListView {
+        clip: true
+        focus: true
+        Layout.preferredHeight: childrenRect.height
+        spacing: attachmentsPopupElementView.mediaMargin
+        model: attachmentsPopupElementView.controller.popupAttachmentItems
+
+        Layout.fillWidth: true
+        Layout.leftMargin: attachmentsPopupElementView.mediaMargin
+        Layout.rightMargin: attachmentsPopupElementView.mediaMargin
+
+        delegate: Item {
+            width: parent.width
+            height: childrenRect.height
+
+            MouseArea {
+                anchors.fill: parent
+                hoverEnabled: true
+
+                onClicked: {
+                    if (!model.listModelData.dataFetched) {
+                        model.listModelData.downloadAttachment();
+                    } else {
+                        if( model.listModelData.popupAttachmentType === QmlEnums.PopupAttachmentTypeImage) {
+                            fullScreenImage.source = model.listModelData.localData;
+                            fullScreenImageDialog.visible = true;
+                        } else {
+                            if (Qt.platform.os !== "android" && Qt.platform.os !== "ios") {
```

Comment:
probably worthy of a comment about the differing behavior and possibly a link to any bugs/enhancements we are tracking, as we ideally can support this on mobile in the future

Reply:
Ah yes. I was waiting to hear back about the thumbnails as well. I'll add a bug link and explanation here.

Reply:
bugs/enhancements added and a comment about their current state.